Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -57,8 +73,8 @@ public ValidationResult GetValidationResult((Contest?, int?, UserInfoModel?, boo
     }
 
     private static bool IsContestExpired(
-        Contest contest,
-        string userId,
+        IContestActivityServiceModel contest,
+        Participant? participant,
```

Comment:
Don't mix entity types and service models. maybe use a participant service model here

Reply:
Added ParticipantServiceModel to use for validations